test/smoke/src/sql/areas/queryEditor/queryEditor.test.ts

```diff
@@ -21,6 +21,8 @@ export function setup() {
 			await app.workbench.connectionDialog.connect();
 			await app.workbench.queryEditor.commandBar.run();
 			await app.workbench.queryEditor.waitForResults();
+
+			await app.workbench.quickaccess.runCommand('workbench.action.revertAndCloseActiveEditor');
```

Comment:
What about the rest of the tests? Probably would be a good idea to add it to openAndExecuteFile as well